src/App.js

```diff
@@ -7,6 +7,7 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
 
 import NavBar from './Components/NavBar';
 import Politicians from './Components/Politicians/Politicians';
+import PoliticianItemContainer from './Components/Politicians/PoliticianItemContainer';
```

Comment:
Please say if Flow is being more of an hindrance than useful.
I'm always happy to help with typings :)

Reply:
Id like to learn Flow so its not a hinderance now